gemrb/core/Game.h

```diff
@@ -476,7 +476,7 @@ class GEM_EXPORT Game : public Scriptable {
 	/** check if the passed actor is a victim of timestop */
 	bool TimeStoppedFor(const Actor* target=NULL);
 	/** updates the infravision info */
-	void Infravision();
+	void Infravision(const Actor *target=NULL);
```

Comment:
this compiles? Renamed parameter.

Reply:
To me and my compilers (both GCC), this looks valid won't issue warnings.